.github/workflows/deploy-frontends-to-staging.yml

```
@@ -338,6 +338,9 @@ jobs:
         run: |
           cd website2/
           docker build . \
+          --build-arg NEXT_PUBLIC_API_URL: ${{ secrets.WEBSITE_STAGE_NEXT_PUBLIC_API_URL }} \
+          --build-arg NEXT_PUBLIC_OPENCAGE_API_KEY: ${{ secrets.WEBSITE_NEXT_PUBLIC_OPENCAGE_API_KEY }} \
+          --build-arg NEXT_PUBLIC_API_TOKEN: ${{ secrets.WEBSITE_STAGE_NEXT_PUBLIC_API_TOKEN }} \          
```

yamllint

[error] trailing spaces

(trailing-spaces)